src/common/api/private-api.ts

```diff
@@ -591,3 +589,31 @@ export const getAnnouncementsData = async (): Promise<Announcement[]> => {
     throw error;
   }
 };
+export interface Recoveries {
+  username: string;
+  email: string;
+  public_keys: Object;
+}
+
+export interface getRecoveriesEmail extends Recoveries {
```

Comment:
Interfaces/classes/consts shouldn't be named as action. Use nouns instead: `GetRecoveriesEmailResponse`